format.go

```diff
@@ -467,3 +467,13 @@ func PrintStatus(w io.Writer, stat *status.Status, formatter Formatter) {
 		}
 	}
 }
+
+// PrintFormattedStatus writes the gRPC status response in its entirety given the
+// provided Formatter
+func PrintFormattedStatus(w io.Writer, stat *status.Status, formatter Formatter) {
```

Comment:
I don't think this function carries its weight: its not worth adding to the exported surface area of the `grpcurl` package. Let's just inline it into the `main` package. The added test can just be removed since it don't really do much beyond existing tests in `format_test.go` anyway.

Reply:
Sounds good, I added right under `dial`.